src/rnpkeys/tui.c

```diff
@@ -2,6 +2,7 @@
 #include "crypto.h"
 #include "crypto/ec.h"
 #include "rnp/rnpcfg.h"
+#include "rnpkeys.h"
```

Comment:
why it is now needed?

Reply:
Declaration of `rnp_generate_key_expert_mode`